src/backend/expungeservice/models/helpers/charge_classifier.py

```diff
@@ -137,7 +155,10 @@ def _subsection_6(section, level, statute):
             "163165",  # ( ineligible if under subection(1)(h) ; Assault in the third degree of a minor 10 years or younger)
         ]
         if section in conditionally_ineligible_statutes:
-            return [Subsection6] + ChargeClassifier._classification_by_level(level, statute)
+            question_string = "TODO: FIXME: Is this charge eligible?"
```

Comment:
"Does this charge meet the eligibility requirements listed in 137.225(6)? "
options = ["Yes", "No"]

Reply:
We eventually want to split it out but I want to get the "framework" down first.